src/components/Header/Header.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react'
+import { useValues, useActions } from 'kea'
+import { Link } from 'gatsby'
+import { layoutLogic } from '../../logic/layoutLogic'
+import whiteLogo from '../../images/posthog-logo-white.svg'
+import darkLogo from '../../images/posthog-logo-150x29.svg'
+
+interface NavbarLinkProps {
+    to?: string
+    href?: string
+    children: any
+    textLight: boolean
+    className?: string
+}
+
+const NavbarLink = ({ to, href, children, textLight, className = '' }: NavbarLinkProps) => {
+    const baseClasses = 'opacity-80 hover:opacity-100 px-4 py-2 font-semibold tracking-wider '.concat(className)
+    const classList = textLight
+        ? `text-white hover:text-white ${baseClasses}`
+        : `text-black hover:text-black ${baseClasses}`
+
+    return (
+        <li className="leading-none">
+            {to ? (
+                <Link to={to} className={classList}>
+                    {children}
+                </Link>
+            ) : (
+                <a href={href} className={classList}>
+                    {children}
+                </a>
+            )}
+        </li>
+    )
+}
+
+const PrimaryCta = ({ children, className = '' }: { children: any; className?: string }) => {
+    const { setIsGetStartedModalOpen } = useActions(layoutLogic)
+
+    const classList = `button-primary ${className}`
+
+    return (
+        <li className="leading-none">
+            <button onClick={() => setIsGetStartedModalOpen(true)} className={classList}>
+                {children}
+            </button>
+        </li>
+    )
+}
```

Comment:
@yakkomajuri I think I'd like to leave this one here for now. It's just a placeholder until we work through the landing page where we'll have a separate button components that we should be using globally. This one was just intended for the CTA in the navbar but because it had to duplicate between horizontal nav, and collapsed mobile nav I figured I'd extract it in the same file so the class names weren't duplicated and likely to get out of sync.

Is that an okay path for you?

Reply:
Fine by me! This was more of a nit than anything